cmd/chihaya/config.go

```diff
@@ -106,6 +107,17 @@ func (cfg ConfigFile) CreateHooks() (preHooks, postHooks []middleware.Hook, err
 				return nil, nil, errors.New("invalid interval variation middleware config: " + err.Error())
 			}
 			preHooks = append(preHooks, hook)
+		case "deniability":
```

Comment:
The more I see this, the more I want to bring back the registered middleware model so we can have a function that looks up a name and returns a 
```go
type PreHookConstructor func([]byte) (middleware.Hook, error)
```